variants/CY8CKIT-062S2-AI/pins_arduino.h

```diff
@@ -193,6 +193,8 @@ const cyhal_gpio_t mapping_gpio_pin[] = {
 
 const uint8_t GPIO_PIN_COUNT = (sizeof(mapping_gpio_pin) / sizeof(mapping_gpio_pin[0])) - 1;
 
+bool gpio_initialized[GPIO_PIN_COUNT] = {false};
```

Comment:
Anything speaking against having this array directly in digital_io.c? 
Then we don´t have to define it for each board.